PWGCF/Flow/Tasks/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ o2physics_add_dpl_workflow(flow-runby-run
                     COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(flow-gfw-pbpb
-                    SOURCES FlowGFWPbPb.cxx
+                    SOURCES flowGFWPbPb.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::GFWCore
                     COMPONENT_NAME Analysis)
```

Comment:
The `linter` is suggesting you which name you should use according to the name of your workflow, `flowGfwPbpb.cxx`. Probably you can reconsider if you want `PbPb` in the name of either your workflow or your source code file. I guess, apart from configurations, nothing should prevent to run your task on pp or even in OO if available